install/installer.class.php

```diff
@@ -494,20 +464,66 @@ public function createNotificationTargetInvitation() {
     *
     * @param string $fromVersion
     */
-   protected function upgrade($fromVersion) {
-      switch ($fromVersion) {
-         case '2.0.0':
-            // Example : upgrade to version 3.0.0
-            // $this->upgradeOneStep('3.0.0');
-         case '3.0.0':
-            // Example : upgrade to version 4.0.0
-            // $this->upgradeOneStep('4.0.0');
+   protected function upgrade(Migration $migration) {
+      $this->migration = $migration;
+      $fromSchemaVersion = $this->getSchemaVersion();
+
+      switch ($fromSchemaVersion) {
+         case '2.0':
+            // Example : upgrade to schema version 3.0
+            // $this->upgradeOneStep('3.0');
+         case '3.0':
+            // Example : upgrade to version 4.0
+            // $this->upgradeOneStep('4.0');
 
          default:
       }
-      if (PluginFlyvemdmCommon::endsWith(PLUGIN_FLYVEMDM_VERSION, "-dev")) {
-         $this->upgradeOneStep('dev');
+
```

Comment:
(strange: the closing bracket is missing in the diff!)

I decided to obsolete upgrade_to_dev.php to avoid upgrade misses. Now we rely only on the schema version for upgrades.